index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="description" content="Social media application for connecting with people">
+        <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
```

Comment:
```suggestion
        <link rel="stylesheet" href="/node_modules/bootstrap/dist/css/bootstrap.min.css">
```
Absolute links are more `portable`, you can copy/paste them anywhere and the route will work.